wdl-lint/tests/lints/missing-comment-space/source.wdl

```diff
@@ -1,12 +1,16 @@
-#@ except: BlankLinesBetweenElements, CommentWhitespace, DescriptionMissing
+#@ except: Todo
+
 ##This preamble comment is missing a space.
-##
-## But this one isn't and neither are the empty ones
+## 
+## TODO: Line1 creates two diagnostics, one for CommentWhitespace and
```

Comment:
File `TODO` issue. I assume here `CommentWhitespace` should ignore preamble comments?